core/modules/server/routes/put-tiddler.js

```diff
@@ -30,6 +30,16 @@ exports.handler = function(request,response,state) {
 	if(fields.revision) {
 		delete fields.revision;
 	}
+	// If this is a skinny tiddler, it means the client never got the full
+	// version of the tiddler to edit. So we must preserve whatever text
+	// already exists on the server, or else we'll inadvertently delete it.
+	if(fields._is_skinny !== undefined) {
+		var tiddler = state.wiki.getTiddler(title);
+		if(tiddler) {
+			fields.text = tiddler.fields.text;
```

Comment:
What about other fields, like `tags` or `tmap.id` ?

Reply:
What *about* them? They are always loaded, regardless of lazy loading. This fix was to prevent body deletion when either of those two fields are added or modified.

Reply:
Oh, I forget that. Then lazy loading won't work for me... I have 10k+ tiddler without text, only calendar event related metadata fields...

Reply:
If they don't have any text, then you shouldn't worry about size anyway, except if you store images in the fields.